ChatApp/ChatView/MessageView.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  MessageView.swift
+//  ChatApp
+//
+//  Created by Giorgi Makadze on 19.04.2023.
+//
+
+import Foundation
+import UIKit
+
+class MessageView: UIView, UITextViewDelegate, UICollectionViewDelegate {
+    private let textView = UITextView() // texfield
```

Comment:
added extension for MessageView and including necessary methods for UITextViewDelegate, UICollectionViewDelegate would be better. like this can separate methods.